services/sentences.js

```diff
@@ -27,119 +22,85 @@ const GLOBAL_LOCATION = {
-
-        // sum over different bound1 within year
-        const summedPlantationsLoss =
-          latestPlantationLoss &&
-          latestPlantationLoss.length &&
-          latestPlantationLoss[0].area
-            ? sumBy(latestPlantationLoss, 'area')
-            : 0;
-        const summedPlantationsEmissions =
-          latestPlantationLoss &&
-          latestPlantationLoss.length &&
-          latestPlantationLoss[0].emissions
-            ? sumBy(latestPlantationLoss, 'emissions')
-            : 0;
-        const summedLoss =
-          latestLoss && latestLoss.length && latestLoss[0].area
-            ? sumBy(latestLoss, 'area')
-            : 0;
-        const summedEmissions =
-          latestLoss && latestLoss.length && latestLoss[0].emissions
-            ? sumBy(latestLoss, 'emissions')
-            : 0;
-
-        const data = {
-          totalArea:
-            extent && extent.length && extent[0].total_area
-              ? sumBy(extent, 'total_area')
-              : 0,
-          extent:
-            extent && extent.length && extent[0].extent
-              ? sumBy(extent, 'extent')
-              : 0,
-          plantationsExtent:
-            plantationsExtent &&
-            plantationsExtent.length &&
-            plantationsExtent[0].extent
-              ? sumBy(plantationsExtent, 'extent')
-              : 0,
-          primaryExtent:
-            primaryExtent && primaryExtent.length && primaryExtent[0].extent
-              ? sumBy(primaryExtent, 'extent')
-              : 0,
-          totalLoss: {
-            area: summedLoss || 0,
-            year: latestYear || 0,
-            emissions: summedEmissions || 0,
-          },
-          plantationsLoss: {
-            area: summedPlantationsLoss || 0,
-            emissions: summedPlantationsEmissions || 0,
-          },
-          primaryLoss:
-            primaryLoss && primaryLoss.length
-              ? reverse(sortBy(primaryLoss, 'year'))[0]
-              : {},
-        };
-
-        return data;
+      umd_tree_cover_loss__year: 2023,
+      isNaturalForest: true,
+    });
+
+    let extent = 0;
+    let totalArea = 0;
+
+    extentNaturalForestResponse.data.data.forEach((item) => {
+      totalArea += item.area__ha;
+
+      if (item.sbtn_natural_forests__class === 'Natural Forest')
+        extent += item.area__ha;
+    });
+
+    let lossArea = 0;
+    let emissions = 0;
+
+    lossNaturalForestResponse.data.data.forEach((item) => {
+      emissions += item.gfw_gross_emissions_co2e_all_gases__mg;
+
+      if (item.sbtn_natural_forests__class === 'Natural Forest') {
+        lossArea += item.area;
       }
-    )
-  );
+    });
+
+    return {
+      totalArea,
+      extent,
+      plantationsExtent: 0,
+      primaryExtent: 0,
+      totalLoss: {
+        area: lossArea,
+        year: 2023,
+        emissions,
+      },
+      plantationsLoss: {
+        area: 0,
+        emissions: 0,
+      },
+      primaryLoss: {},
+    };
```

Comment:
looks like the dashboard sentences only require the following:
- totalArea
- extent
- totalLoss: {
  area,
  year,
  emissions,
}

hence I left the others with 0